packages/kbn-grid-layout/grid/grid_layout.tsx

```diff
@@ -121,11 +121,6 @@ export const GridLayout = ({
           rowIndex={rowIndex}
           renderPanelContents={renderPanelContents}
           gridLayoutStateManager={gridLayoutStateManager}
-          toggleIsCollapsed={() => {
```

Comment:
I think we can collocate this inside row, so I moved it there.

Reply:
Nice cleanup 🚀